config/routes.rb

```diff
@@ -2416,6 +2416,20 @@
         save_post
     },
 
+    :volume_mapping => {
+      :get  => %w[
+        download_data
+        download_summary_pdf
+        index
+        perf_top_chart
```

Comment:
some of these routes are not required for this PR, can you please clean these up

Reply:
I think I now have the correct minimal set of routes? I wasn't sure about `compare_get`?